modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -92,6 +92,135 @@ using the `Flex` component and to replace the `spacing` prop with the `gap` prop
 > **Note**: If you are consuming `StackProps` for a custom component, please make sure to change the
 > `spacing` prop to the `gap` prop
 
+---
+
+### Menu in preview-react
+
+We've removed the `Menu` component that was located in `preview-react` and now will be replaced with
+the `Menu` component in `main`. Please see the `Menu` documentation
```

Comment:
yeah im not too sure here, if we want to say `main` or just Main. with no back tickets, because we don't have a package named `main` it's `@workday/canvas-kit-react`

Reply:
I'll defer to @jamesfan